px-map-behavior-popup.es6.js

```diff
@@ -247,8 +247,9 @@
 
       const maxWidth = 400;
       const minWidth = 300;
+      const maxHeight = 300;
```

Comment:
You are right, we can make this as part of the attribute/property of the px-map-marker-group component, and other individual popup components, do you want me to work on it.